Sources/SwiftGodot/Core/GArrayCollection.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  GArrayCollection.swift
+//
+//
+//  Created by Estevan Hernandez on 11/28/23.
+//
+
+@_implementationOnly import GDExtension
```

Comment:
not sure if this line is actually needed or not, it seems some compiler errors may have gone away by adding it. The error was unknown type`GArrayCollection`